std/experimental/allocator/building_blocks/bitmapped_block.d

```diff
@@ -50,6 +50,7 @@ struct BitmappedBlock(size_t theBlockSize, uint theAlignment = platformAlignment
     import std.typecons : Ternary;
     import std.typecons : tuple, Tuple;
 
+    version(StdUnittest)
```

Comment:
why do we need this here? I'd assume if no unittest is conducted, unittests are ignored anyway.

Reply:
From DIP82:

> At present, unittest blocks inside of a templated type are compiled into every instantation, which only works with tests that are generic, which most tests aren't, and it almost never works with examples and thus doesn't work for ddoc-ed unittest blocks.

1) these unittest blocks might not even be run (I have a couple of these issues during the runnable examples story)
2) these unittest blocks will be executed as part of a user's testsuite